### PR TITLE
[depends] remove no more needed libmp3lame

### PR DESCRIPTION
**Pull request by AlwinEsch**
Found during creation of my multi platform system a fault with libmp3lame on i386 system which becomes called from ./tools/buildsteps/linux32/make-depends.

Also reported here on debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=760047

Has included the needed fix patches to the addon on https://github.com/xbmc/audioencoder.lame/pull/10 and can be removed now here as dependency.

### CODE REVIEW

**Comment by FernetMenta** [comment]
why not dropping this lib? if I am not mistaken it is not used by the application.

**Comment by AlwinEsch** [comment]
Has not direct reviewed the use of them, you are right not used anymore inside kodi itself.

I check the audioencoder.lame add-on build now without the library present.

**Comment by FernetMenta** [comment]
> I check the audioencoder.lame add-on build now without the library present.

if this addon requires the lib, it should be moved to the addon

**Comment by stefansaraev** [comment]
it's already moved. 
https://github.com/xbmc/audioencoder.lame/blob/master/depends/common/lame/CMakeLists.txt

libmp3lame should be safe to be removed from depends

**Comment by AlwinEsch** [comment]
Is updated and libmp3lame removed from depends.
The previous patches are now included on add-on itself.

jenkins build this please.

File: project/cmake/addons/bootstrap/repositories/binary-addons.txt [hunk 1]
```diff
@@ -1 +1 @@
-binary-addons https://github.com/xbmc/repo-binary-addons.git master
\ No newline at end of file
+binary-addons https://github.com/AlwinEsch/repo-binary-addons.git master
```
Comment (wsnipex) [inline project/cmake/addons/bootstrap/repositories/binary-addons.txt]: please remove this

Comment (MartijnKaijser) [inline project/cmake/addons/bootstrap/repositories/binary-addons.txt]: @wsnipex if you read the PR it's a temp commit for testing

Comment (wsnipex) [inline project/cmake/addons/bootstrap/repositories/binary-addons.txt]: and its been tested and is ready for merge as soon as this is removed...

**Comment by AlwinEsch** [comment]
Is removed

**Pull request merged**